TrainingExtensions/tensorflow/src/python/aimet_tensorflow/keras/batch_norm_fold.py

```diff
@@ -313,8 +313,8 @@ def _get_weight_tensor_transpose_reshape(conv_linear: LAYER_TYPE) -> libpymo.Ten
     weight_tensor = libpymo.TensorParams()
 
     # linear array to be sent for bn fold
-    weight = conv_linear.kernel.numpy()
-    shape = conv_linear.kernel.shape
+    weight = conv_linear.get_weights()[0]
```

Comment:
What is 0 here? what does get_weights() return?

Reply:
Hi @quic-akhobare 
Return of [get_weights()](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer#get_weights) depends on layer type.

If layer is Conv2D with bias, it will return `[weight_tensor, bias_tensor]`
else if layer is Conv2D without bias, it will return `[weight_tensor]`
else if layer is BatchNorm, it will return `[gamma, beta, moving_mean, moving_variance]`

I feel the name of get_weights is ambiguous, but it's implementation of TensorFlow :(

Reply:
Got it. PyTorch calls these weights too.. E.g. gamma maps to "weight" for PyTorch BN layers